fluss-server/src/test/java/com/alibaba/fluss/server/kv/rocksdb/RocksDBResourceContainerTest.java

```diff
@@ -187,8 +193,237 @@ void testConfigurationOptionsFromConfig() throws Exception {
+        // Statistics should be enabled by default
+        try (RocksDBResourceContainer container = new RocksDBResourceContainer(config, null)) {
+            DBOptions dbOptions = container.getDbOptions();
+            assertThat(dbOptions.statistics()).isNotNull();
+            assertThat(container.getStatistics()).isNotNull();
+        }
+    }
+
+    @Test
+    void testStatisticsDisabled() throws Exception {
+        Configuration config = new Configuration();
+        config.setBoolean(ConfigOptions.KV_STATISTICS_ENABLED.key(), false);
+
+        try (RocksDBResourceContainer container = new RocksDBResourceContainer(config, null)) {
+            DBOptions dbOptions = container.getDbOptions();
+            // Statistics should be null when disabled
+            assertThat(dbOptions.statistics()).isNull();
+            assertThat(container.getStatistics()).isNull();
+        }
+    }
+
+    @Test
+    void testStatisticsExplicitlyEnabled() throws Exception {
+        Configuration config = new Configuration();
+        config.setBoolean(ConfigOptions.KV_STATISTICS_ENABLED.key(), true);
+
+        try (RocksDBResourceContainer container = new RocksDBResourceContainer(config, null)) {
+            DBOptions dbOptions = container.getDbOptions();
+            assertThat(dbOptions.statistics()).isNotNull();
+            assertThat(container.getStatistics()).isNotNull();
+        }
+    }
+
+    @Test
+    void testStatisticsClosedWithContainer() throws Exception {
+        Configuration config = new Configuration();
+        config.setBoolean(ConfigOptions.KV_STATISTICS_ENABLED.key(), true);
+
+        Statistics statistics;
+        try (RocksDBResourceContainer container = new RocksDBResourceContainer(config, null)) {
+            statistics = container.getStatistics();
+            assertThat(statistics).isNotNull();
+            assertThat(statistics.isOwningHandle()).isTrue();
+        }
+
+        // Statistics should be closed when container is closed
+        assertThat(statistics.isOwningHandle()).isFalse();
+    }
+
+    @Test
+    void testMultipleStatisticsObjectsClosedProperly() throws Exception {
+        Configuration config = new Configuration();
+        config.setBoolean(ConfigOptions.KV_STATISTICS_ENABLED.key(), true);
+
+        List<Statistics> statisticsList = new ArrayList<>();
+
+        // Create multiple containers and collect statistics
+        for (int i = 0; i < 5; i++) {
+            RocksDBResourceContainer container = new RocksDBResourceContainer(config, null);
+            Statistics stats = container.getStatistics();
+            if (stats != null) {
+                statisticsList.add(stats);
+            }
+            container.close();
+        }
+
+        // All statistics should be closed
+        for (Statistics stats : statisticsList) {
+            assertThat(stats.isOwningHandle()).isFalse();
+        }
+    }
+
+    @Test
+    void testStatisticsWithNullInstancePath() throws Exception {
```

Comment:
I don't see any values of this test, what's the use case with a null instance path?